app/models/work_version.rb

```diff
@@ -36,7 +36,6 @@ class WorkTypeUpdateError < RuntimeError; end
   LINK_TEXT = ':link will be inserted here automatically when available:'
 
   after_update_commit -> { work.broadcast_update }
-  after_update_commit -> { work.collection.broadcast_update_collection_summary }
```

Comment:
We no longer update the whole collection summary at once, because depositor users don't see all the stuff in the summary.